static/app/views/settings/account/notifications/utils.tsx

```diff
@@ -41,22 +41,21 @@ export function getDocsLinkForEventType(
   event: DataCategoryExact | string // TODO(isabella): get rid of strings after removing need for backward compatibility on gs
 ) {
   switch (event) {
-    case DataCategoryExact.TRANSACTION || 'transaction':
+    case DataCategoryExact.TRANSACTION:
       // For pre-AM3 plans prior to June 11th, 2024
       return 'https://docs.sentry.io/pricing/quotas/legacy-manage-transaction-quota/';
-    case DataCategoryExact.SPAN ||
-      DataCategoryExact.SPAN_INDEXED ||
-      'span' ||
-      'span_indexed':
```

Comment:
This wasn't working before when the function was called with `DataCategoryExact.SPAN_INDEXED` or `span_indexed`. 

The case would boil down to be the same as `case DataCategoryExact.SPAN` only so values like "span_indexed" and "spanIndexed" wouldn't match at all and would fall through to the default case below.

Can play with it here: https://www.typescriptlang.org/play/?#code/GYVwdgxgLglg9mABFOA5EBbARgUwE4AUMYADiFAFyIDOUexA5gJSIDeAUO4jQO4xQQAFkVLkWHbtwgBDajkQByBDgUUuk7nhxQQeJAEYA3Jw0y5iqDzgLEAH1uKAJnGqr1GrTr2IATMfeIjjjA0iAANpQBmtq6BgAMccbcAL7sqewQCNRwYTgAdGFwDARKYCoANMhomLiEpSpMTMaZYNm5BUUlltaVKOjY+F1WCo3NWTn5hcUKzq691QN1syNNQA

<img width="884" alt="SCR-20250110-mqxy" src="https://github.com/user-attachments/assets/f9a09ff7-2c8a-46cb-b456-d738eb13983e" />

Reply:
cc @isabellaenriquez